spinn_front_end_common/interface/interface_functions/insert_extra_monitor_vertices_to_graphs.py

```diff
@@ -129,7 +129,8 @@ def _handle_data_extraction_vertices(
             # add to application graph if possible
             if application_graph is not None:
                 equiv_vertex = self._exists_equiv_vertex(
-                    ethernet_chip.x, ethernet_chip.y, application_graph,
+                    ethernet_chip.x, ethernet_chip.y,
+                    vertex_to_ethernet_connected_chip_mapping,
                     DataSpeedUpPacketGather)
```

Comment:
remove as you dont need the type now
```suggestion
                    )
```